pkgs/top-level/all-packages.nix

```
@@ -1915,6 +1915,8 @@ with pkgs;
 
   passExtensions = recurseIntoAttrs pass.extensions;
 
+  inherd-quake = callPackage ../applications/misc/inherd-quake { };
```

We commonly put this in top-level.

```suggestion
  inherd-quake = callPackage ../applications/misc/inherd-quake {
    inherit (darwin.apple_sdk.frameworks) CoreServices Security;
  };
```